exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,11 @@
 // Declare your function here
 
+function createLongGreeting(a, b) {
+  return `Hello, my name is ${a}, and I am ${b} years old`;
+}
+
+
+
 const greeting = createLongGreeting("Daniel", 30);
 
 console.log(greeting);
```

Comment:
Nice! I think it would be better to  use semantic parameters for the function, like name and age instead of a and b.